{"links": {"self": {"href": "data/repositories/labscript_suite/blacs/pullrequests/68/comments/100229558.json"}, "html": {"href": "#!/labscript_suite/blacs/pull-requests/68/_/diff#comment-100229558"}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 68, "links": {"self": {"href": "data/repositories/labscript_suite/blacs/pullrequests/68.json"}, "html": {"href": "#!/labscript_suite/blacs/pull-requests/68"}}, "title": "Handle authentication error connecting to lyse"}, "content": {"raw": "Thanks for testing!\n\nI\u2019m not able to reproduce the problem. Is the BLACS computer on a new enough version of zprocess? 2.12.5 is required for ordinary zmq\\_get calls to have the possibility of raising AuthenticationFailure - I only added that the other day and didn\u2019t make anything depend on it since it\u2019s not strictly required, so that might be why you\u2019re not seeing it.\n\nMy test setup is the following, with a bit of hoop-jumping since I\u2019m only on one computer:\n\n* Comment out shared secret in labconfig\n* kill zlock and zlog if running\n* start lyse\n* Kill the unencrypted zlock and zlog servers that lyse started\n* Uncomment shared secret\n* Start BLACS\n* See authentication failure message in tooltip\n\nSame thing happens if I have lyse be the encrypted one and BLACS be the unencrypted one.", "markup": "markdown", "html": "<p>Thanks for testing!</p>\n<p>I\u2019m not able to reproduce the problem. Is the BLACS computer on a new enough version of zprocess? 2.12.5 is required for ordinary zmq_get calls to have the possibility of raising AuthenticationFailure - I only added that the other day and didn\u2019t make anything depend on it since it\u2019s not strictly required, so that might be why you\u2019re not seeing it.</p>\n<p>My test setup is the following, with a bit of hoop-jumping since I\u2019m only on one computer:</p>\n<ul>\n<li>Comment out shared secret in labconfig</li>\n<li>kill zlock and zlog if running</li>\n<li>start lyse</li>\n<li>Kill the unencrypted zlock and zlog servers that lyse started</li>\n<li>Uncomment shared secret</li>\n<li>Start BLACS</li>\n<li>See authentication failure message in tooltip</li>\n</ul>\n<p>Same thing happens if I have lyse be the encrypted one and BLACS be the unencrypted one.</p>", "type": "rendered"}, "created_on": "2019-04-28T14:57:02.861356+00:00", "user": {"display_name": "Chris Billington", "uuid": "{e363c5a9-5075-4656-afb5-88bd6a6dceeb}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Be363c5a9-5075-4656-afb5-88bd6a6dceeb%7D"}, "html": {"href": "https://bitbucket.org/%7Be363c5a9-5075-4656-afb5-88bd6a6dceeb%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/9238baf7300c41c0e7294db922899e6ad=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsCB-1.png"}}, "nickname": "cbillington", "type": "user", "account_id": "557058:cbf1bc43-1dc2-477b-9e25-1a8f40fd7ee3"}, "updated_on": "2019-04-28T14:57:11.297019+00:00", "type": "pullrequest_comment", "id": 100229558}